keyboards/etiennecollin/wave/info.json

```diff
@@ -0,0 +1,109 @@
+{
+    "keyboard_name": "Wave",
+    "maintainer": "etiennecollin",
+    "manufacturer": "Etienne Collin",
+    "url": "https://github.com/etiennecollin/wave",
+    "bootloader_instructions": "Press the button marked RESET on the PCB or bridge the pins marked RST and GND on the Elite-C",
+    "tags": ["ortho", "split"],
+
+    "bootloader": "atmel-dfu",
+    "build": {
+        "lto": true
+    },
+    "development_board": "elite_c",
+    "pin_compatible": "elite_c",
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "0.0.1",
+        "vid": "0x6967",
+        "pid": "0x0000"
+    },
+
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+
+    "matrix_pins": {
+        "direct": [
+            ["C6", "D4", "D0", "D1", "D3"],
+            ["B6", "B5", "B4", "E6", "D7"],
+            ["F6", "F7", "B1", "B3", "B2"],
+            ["NO_PIN", "NO_PIN", "F5", "F4", "B0"]
+        ]
+    },
+    "community_layouts": ["split_3x5_3"],
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D2",
+        "bootmagic": {
+            "matrix": [0, 0]
+        },
```

Comment:
```suggestion
```

Reply:
Why should this be removed? Is the default behaviour to have bootmagic on `[0,0]`?

Reply:
Done